sdk/tests/conformance/textures/misc/texture-video-transparent.html

```diff
@@ -92,21 +92,22 @@
     const tolerance = 5;
     const red = [255, 0, 0];
     const green = [0, 255, 0];
-    const blue = [0, 0, 255];
+    const blue = [0, 0, 200];
```

Comment:
Did you mean to make this change, or was it only for testing?

Reply:
that was meant to be deleted. BTW I'm extending to print the whole buf.